src/main/kotlin/com/vk/admstorm/startup/ChangeSshBackendStartup.kt

```diff
@@ -10,44 +10,52 @@ import com.vk.admstorm.notifications.AdmNotification
 object ChangeSshBackendStartup {
     private val LOG = logger<ChangeSshBackendStartup>()
 
-    private const val SSH_NOTIFICATOR_OPTION = "sshLegacyChanges"
+    private const val LEGACY_SSH_NOTIFICATOR_OPTION = "sshLegacyChanges"
+    private const val SSH_NOTIFICATOR_OPTION = "sshOpenSSHChanges"
     private const val SSH_CONFIG_BACKEND = "ssh.config.backend"
 
     private fun changeSshConfiguration(configValue: SshConnectionConfigService.Kind) {
         AdvancedSettings.setEnum(SSH_CONFIG_BACKEND, configValue)
     }
 
     fun changeConfigurationProcess(project: Project) {
-        val dntShow = PropertiesComponent.getInstance(project).getBoolean(SSH_NOTIFICATOR_OPTION)
+        val prpCmp = PropertiesComponent.getInstance(project)
```

Comment:
What is `prpCmp`? Why not `propertiesComponent`, `properties`, `prop`? (I would use 'properties`)